cursorless-talon/src/actions/actions.py

```diff
@@ -21,10 +21,10 @@
         "{user.cursorless_custom_action}"
     )
 )
-def cursorless_action_or_vscode_command(m) -> dict:
+def cursorless_action_or_server_command(m) -> dict:
```

Comment:
I'm not sure `server_command` is the right name for all these.  Maybe `ide_command`?

Reply:
Dopne